index.html

```diff
@@ -6,11 +6,32 @@
     <meta charset="UTF-8">
```

Comment:
I was wondering why the media queries weren't working when using Chrome's responsive mode, then I noticed a missing meta tag. I'd suggest adding this viewport meta tag so that the page scales properly: 
`<meta name="viewport" content="width=device-width, initial-scale=1, minimum-scale=1" />`

Here's a short link explaining how this helps the page scale appropriately: https://www.w3schools.com/css/css_rwd_viewport.asp

Reply:
I didn't know this. Will update it in the next commit. And thank you for the link! Appreciate it